presentation/src/main/java/com/anytypeio/anytype/presentation/editor/EditorViewModel.kt

```diff
@@ -3160,7 +3184,13 @@ class EditorViewModel(
                 }
             }
             ObjectType.Layout.SET, ObjectType.Layout.COLLECTION -> {
-                proceedWithOpeningDataViewObject(target = target)
+                val space = wrapper.spaceId
+                if (space != null) {
+                    proceedWithOpeningDataViewObject(
+                        target = target,
+                        space = SpaceId(checkNotNull(wrapper.spaceId))
```

Comment:
It seems that deliberately crashing the application is not the best option for the user in this case, better through Timber.e to proceed